compass/package-lock.json

```diff
@@ -1,8 +1,594 @@
 {
   "name": "@kyma-project/compass-ui",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I am not sure for this change. 
Look like you have upgraded npm tool that produces never version of lock file and to be honest do not know what is the implication, as we are running the fixed nodes version in builder container that may not support tis file format.